### ISSUE TITLE
Dropout: opposite meaning?

### ISSUE THREAD

**Issue by seann999** [hassony2/kinetics_i3d_pytorch#6]
Thanks for the great repo!

In the [code](https://github.com/hassony2/kinetics_i3d_pytorch/blob/master/src/i3dpt.py#L165) it says `dropout_keep_prob=1` but in the PyTorch [documentation](http://pytorch.org/docs/0.3.0/nn.html?highlight=dropout#torch.nn.Dropout) `p` is the probability of an element to be zeroed.

**Comment by hassony2** [comment]
Hi @seann999 

Thank you very much for pointing this out !
This is fixed by #7.
